src/devices/Gpio/Drivers/OrangePiLite2Driver.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+
+namespace Iot.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// A GPIO driver for the Orange Pi Lite 2.
+    /// </summary>
+    /// <remarks>
+    /// SoC: Allwinner H6 (sun50iw6p1)
+    /// </remarks>
+    public class OrangePiLite2Driver : Sun50iw6p1Driver
+    {
+        private static readonly int[] _pinNumberConverter = new int[]
+        {
+            -1, -1, -1, MapPinNumber('H', 6), -1, MapPinNumber('H', 5), -1, MapPinNumber('H', 4), MapPinNumber('D', 21), -1,
+            MapPinNumber('D', 22), MapPinNumber('D', 24), MapPinNumber('C', 9), MapPinNumber('D', 23), -1, MapPinNumber('D', 26),
+            MapPinNumber('C', 8), -1, MapPinNumber('C', 7), MapPinNumber('C', 2), -1, MapPinNumber('C', 3), MapPinNumber('D', 25),
+            MapPinNumber('C', 0), MapPinNumber('C', 5), -1, MapPinNumber('H', 3)
+        };
+
+        /// <inheritdoc/>
+        protected override int PinCount => _pinNumberConverter.Count(n => n != -1);
```

Comment:
consider caching this value

Reply:
_pinNumberConverter will never be null here, I think there should be no exceptions thrown here.

Reply:
that's not what I meant, I meant that you need to iterate through the list every time you want to check the PinCount and this value is actually constant (`=>` means this will compute every time you access the property, `{ get; } = ...;` would compute it only once)

Reply:
Haha, OK, I see what you mean. Actually, I am too lazy to count the number of pins. I'll change it.